buildkite/pipeline.yml

```diff
@@ -6,17 +6,22 @@ steps:
 
   - label: ":hammer: Unit tests"
     command: docker/test_check
+    key: "unit-tests"
 
   - label: ":hammer_and_pick: Integration tests"
     command: docker/test_integration
+    key: "integration-tests"
 
   - label: ":docker: Docker connection test"
     command: docker/test_connection
+    key: "connection-tests"
 
-  - wait
+  - label: ":codecov: Codecov"
```

Comment:
is it worth trying forcing a failure here (easily done by `command: /bin/false`), to make sure we get pushed images + failed build?

Reply:
Have done that confirmed it has worked see https://buildkite.com/mrc-ide/hintr/builds/1132#_

Reverted that and force pushed to remove from history